driver/flambda_backend_args.ml

```diff
@@ -388,7 +388,7 @@ let mk_dflexpect f =
     \     whose basename matches that of the input .ml file (Flambda 2 only)"
 ;;
 
-let mk_dclosure_offsets f =
+let mk_dslot_offsets f =
   "-dclosure-offsets", Arg.Unit f, " Dump closure offsets (Flambda 2 only)"
```

Comment:
I think the name of the option and the actual flag should be kept in sync.

Reply:
Fixed, that was the intention in fact...